### PR TITLE
Address reviewer comments

### PR DESCRIPTION
**Pull request by LewisAJones**
Hi all, I've implemented a couple of changes to address the reviewers' comments on our manuscript. If one of you could take a quick look when you have a moment and let me know if you think anything else should be included here before I do another release to CRAN (apart from new version number and updating NEWs).

Thanks all!

### CODE REVIEW

File: vignettes/geologic-map.Rmd [hunk 1]
```diff
@@ -272,7 +277,21 @@ Nice! Finally, we can add a few other touches to make it a publication-worthy fi
...
+
+``` r
 library(cowplot)
+```
+
+```
+## Error in library(cowplot): there is no package called 'cowplot'
+```
+
+``` r
```
Comment (willgearty) [inline vignettes/geologic-map.Rmd:294]: 👀

File: vignettes/geologic-map.Rmd [hunk 2]
```diff
@@ -309,16 +328,22 @@ ed_hc_map <- ggplot() +
               size = 0.8, alpha = 1, color = "black") +
   labs(x = "Longitude (º)", y = "Latitude (º)") +
   theme_bw()
+```
+
+```
+## Error in annotation_north_arrow(location = "br", pad_y = unit(0.75, "cm"), : could not find function "annotation_north_arrow"
+```
+
+``` r
```
Comment (willgearty) [inline vignettes/geologic-map.Rmd:337]: 👀

File: vignettes/geologic-map.Rmd [hunk 3]
```diff
@@ -309,16 +328,22 @@ ed_hc_map <- ggplot() +
...
   draw_plot(inset, 0.08, 0.63, width = 0.2, height = 0.2)
 ```
 
-<div class="figure">
-<img src="add-inlay-scale-1.png" alt="plot of chunk add-inlay-scale" width="100%" />
-<p class="caption">plot of chunk add-inlay-scale</p>
-</div>
+```
+## Error in ggdraw(): could not find function "ggdraw"
+```
```
Comment (willgearty) [inline vignettes/geologic-map.Rmd:346]: 👀

File: vignettes/geological-completeness.Rmd [hunk 4]
```diff
@@ -70,7 +71,7 @@ citation("rmacrostrat")
...
+Quantifying the spatiotemporal distribution of sedimentary lithostratigraphic units is fundamental to understanding how environments, and the biodiversity within them, have evolved throughout Earth's history. Previous work by [Peters & Heim (2010)](https://www.jstor.org/stable/25609444) focused on *"the geological completeness of paleontological sampling in North America"*, with the aim of recognizing and overcoming geologically-controlled sampling biases. One component of this article involved quantifying how the number and proportion of lithostratigraphic units preserving different paleoenvironment types (e.g., marine, marginal, terrestrial) varies through time. In this vignette, we will revisit this question and examine how the number and proportion of different sedimentary lithostratigraphic units--grouped by paleoenvironment type--changes over the Phanerozoic. To do so, we will make use of the `rmacrostrat` package to fetch data from the [Macrostrat](https://macrostrat.org) database.
```
Comment (willgearty) [inline vignettes/geological-completeness.Rmd:74]: Should we also update the (admittedly already very long) vignette title, too?

File: vignettes/geological-completeness.Rmd [hunk 5]
```diff
@@ -202,74 +203,23 @@ any(table(litho_units$unit_id) > 1)
...
+## list()
 ```
 
-```
-## 
-##         igneous     metaigneous     metamorphic metasedimentary    metavolcanic        plutonic        volcanic 
-##              51               4             484             301              19            1005            1285
-```
-
-Looks like it's all igneous and metamorphic rocks. That makes sense, it is often very difficult to assign units with lithologies of those classes to environments without more information. Let's add these unknown units to our ongoing data.frame of units for later:
```
Comment (willgearty) [inline vignettes/geological-completeness.Rmd:265]: 😭 😛

File: vignettes/geological-completeness.Rmd [hunk 6]
```diff
@@ -202,74 +203,23 @@ any(table(litho_units$unit_id) > 1)
...
-
-```
-# Combine dataframes
-litho_units <- rbind.data.frame(litho_units, unknown_units)
-```
-
-Great! But we've just remembered that [Macrostrat](https://macrostrat.org) doesn't only cover North America, and we've pulled all units available worldwide. To make a closer reproduction of Figure 14 from [Peters & Heim (2010)](https://www.jstor.org/stable/25609444), we ought to just focus on North America. Conveniently, [Macrostrat](https://macrostrat.org) is split into different projects, which tend to cover different regions:
+Wow, updates to Macrostrat have led to no sedimentary units having an "unknown" paleoenvironment - great! But we've just remembered that [Macrostrat](https://macrostrat.org) doesn't only cover North America, and we've pulled all units available worldwide. To make a closer reproduction of Figure 14 from [Peters & Heim (2010)](https://www.jstor.org/stable/25609444), we ought to just focus on North America. Conveniently, [Macrostrat](https://macrostrat.org) is split into different projects, which tend to cover different regions:
```
Comment (willgearty) [inline vignettes/geological-completeness.Rmd:222]: Love it!

**Review by willgearty — CHANGES_REQUESTED** [review]
Just a few comments! It's looks like you might need to install some packages and recompile the vignettes. Otherwise, looks great, thanks @LewisAJones!

File: vignettes/geological-completeness.Rmd (hunk 4 above)
Comment (LewisAJones) [reply vignettes/geological-completeness.Rmd:74]: I think it is fine given the title of Peters & Heim (2010)

**Comment by LewisAJones** [comment]
Thanks for the quick review @willgearty, let me know if you have any other comments!

**Review by willgearty — APPROVED** [review]

**Pull request merged**